### PR TITLE
[bug 1232435] Add hb healthcheck view

### PR DESCRIPTION
**Pull request by willkg**
This adds an analyzer view that shows all the hb healthchecks and lets
you email the results to the mailing list. Further, it shows you the
recipients for the mailing list making it a little easier to make sure
things are set up correctly.

### CODE REVIEW

**Comment by willkg** [comment]
I need this to test out the healthcheck stuff in production, so I'm going to wait for Travis to greenlight it and then land and push it all out.

**Comment by willkg** [comment]
Green!

File: fjord/heartbeat/analyzer_views.py [hunk 1]
```diff
@@ -9,8 +9,36 @@
...
+)
 from fjord.heartbeat.models import Answer, Survey
 from fjord.journal.models import Record
+from fjord.mailinglist.utils import get_recipients
+
+
+@check_new_user
+@analyzer_required
+def hb_healthcheck(request):
+    """View for viewing healtchecks and kicking off a healtcheck email"""
```
Comment (Osmose) [inline fjord/heartbeat/analyzer_views.py]: healthcheck

Comment (willkg) [inline fjord/heartbeat/analyzer_views.py]: For some reason, i have a hell of a time typing "healthcheck".

**Pull request merged**